ush/ioda/bufr2ioda/utils.py

```diff
@@ -0,0 +1,87 @@
+import netCDF4 as nc
+import numpy as np
+import time
+
+
+def nc_merge(file_name1, file_name2, target_name):

```

Comment:
This is fine for now, but feel strongly this is something that a compiled IODA OOPS app should do.